### PR TITLE
Add a working BGP page

### PR DESCRIPTION
**Pull request by AnuragThePathak**
Screen Shot 2022-08-23 at 9 08 27 PM

https://github.com/openelb/openelb/issues/244

### CODE REVIEW

File: src/utils/object.mapper.js [hunk 1]
```diff
@@ -34,6 +35,30 @@ const getBaseInfo = item => ({
 
 const BGPMapper = item => ({
   ...getBaseInfo(item),
+  status: get(item,
+    `status.nodesPeerStatus.${Object.keys(item?.status.nodesPeerStatus)[0]}.peerState.sessionState`, ""),
```
Comment (weili520) [inline src/utils/object.mapper.js]: We can pull out the `Object.keys` part of the code, like: `const nodeField = Object.keys(item?.status.nodesPeerStatus)[0]`, and use nodeField variable to instead it.

File: src/pages/EIP/List/index.jsx [hunk 2]
```diff
@@ -57,14 +56,14 @@ class List extends Component {
             <Icon name='eip-duotone' size={40} />
             <div className={styles.nametexts}>
               <div className={styles.name}>
-                <text>{name}</text>
+                <p>{name}</p>
                 {isDefault ? <div>
-                  <text>{isDefault}</text>
+                  <p>{isDefault}</p>
                 </div> : ''}
```
Comment (weili520) [inline src/pages/EIP/List/index.jsx]: In react, multi-line html elements, wrapped in parentheses are better. Eg: 
```
(
   <div>
       <span>Text<span>
   <div>
)
```

**Review by weili520 — COMMENTED** [review]
👍

File: src/pages/EIP/List/index.jsx (hunk 2 above)
Comment (AnuragThePathak) [reply src/pages/EIP/List/index.jsx]: Am I required to use parentheses here as well? If so ig we'll modify many part of the codebase.

Comment (weili520) [reply src/pages/EIP/List/index.jsx]: Yes, but you can modify only the parts of this pr. We can later modify this type of issue specifically.

Comment (AnuragThePathak) [reply src/pages/EIP/List/index.jsx]: Cool.

**Pull request merged**